src/components/Apply/Company/CompanyApplicationUtils.js

```diff
@@ -19,6 +17,34 @@ export const generateValidationRule = validationRulesGenerator(CompanyApplicatio
 const HumanReadableErrors = Object.freeze({
     "email-already-exists": "The provided email is already associated to our platform.",
     "company-application-duplicate-email": "There is already an application associated with that email.",
+    "company-application-created-recently": "There is an application created less than 10 minutes ago associated with this email.",
 });
 
+const ValidationErrors = Object.freeze({
+    "invalid-token": {
+        title: "Error! Application does not exist!",
+        text: "An error has occured while validating your application! The application you are trying to validate does not exist,",
+    },
+    "expired-token": {
+        title: "Error! Link has expired!",
+        text: "An error has occured while validating your application! The link sent to you has expired, \
+        you now need to create a new application,",
+    },
```

Comment:
Now that I'm thinking about this, an expired token should be considered as a NOT FOUND application. This is more backend wise

Reply:
Don't use a title, put these in the HumanReadableErrors

Reply:
Our ideia was to make it easier for the company to identify what went wrong, that way they can see if the token has expired or if the token is just wrong, otherwise it will seem as it is the token is wrong when in reality it has only expired.
Okok we will change that